docs/tutorials/openshift.md

```diff
@@ -4,46 +4,34 @@ It is meant to supplement the other provider-specific setup tutorials.
 
 ### For OCP 4.x
 
-In OCP 4.x, if you have multiple ingress controllers then you must specify an ingress controller name or a router name(you can get it from the route's Status.Ingress.RouterName field).
-If you don't specify an ingress controller's or router name when you have multiple ingresscontrollers in your environment then the route gets populated with multiple entries of router canonical hostnames which causes external dns to create a CNAME record with multiple router canonical hostnames pointing to the route host which is a violation of RFC 1912 and is not allowed by Cloud Providers which leads to failure of record creation.
-Once you specify the ingresscontroller or router name then that will be matched by the external-dns and the router canonical hostname corresponding to this routerName(which is present in route's Status.Ingress.RouterName field) is selected and a CNAME record of this route host pointing to this router canonical hostname is created.
-
-Your externaldns CR shall be created as per the following example.
-Replace names in the domain section and zone ID as per your environment.
-This is example is for AWS environment.
+In OCP 4.x, if you have multiple ingress controllers then you must specify an ingress controller name or a router name, you can get it from the route's `status.ingress[*].routerName` field.
```

Comment:
Ingress controller in an OpenShift context is somewhat different to a Kubernetes ingress controller. Can you change it to `IngressController` and link the API in there to indicate what you mean.

Reply:
Link added.